app/views/quick_search/pages/home.html.erb

```diff
@@ -38,6 +38,16 @@
     </div>
   </div>
 
+  <% if Settings.ENABLED_SEARCHERS.include?('exhibits') %>
+    <div class='card'>
+      <%= image_tag 'exhibits.jpg', class: 'card-img-top', alt: '', role: 'presentation' %>
```

Comment:
@mejackreed I ran this through a basic jpg compressor, but if you have some compression-fu you would like to share, feel free!

Reply:
I think this is fine, especially at 14KB. The other png was at like 600KB and was noticeable.